pkg/virt-handler/heartbeat/ksm.go

```diff
@@ -286,19 +287,19 @@ func handleKSM(node *v1.Node, clusterConfig *virtconfig.ClusterConfig) (bool, bo
 	sleepMsBaseline = uint64(getIntParam(node, kubevirtv1.KSMSleepMsBaselineOverride, sleepMsBaselineDefault, 1, math.MaxInt))
```

Comment:
I wonder why we set all these vars are global wouldn't it make more sense to set them as local vars in `calculateNewRunSleepAndPages` but i guess it's not related to this PR

Reply:
https://github.com/kubevirt/kubevirt/pull/11334